services/flow-repository/app/api/controllers/startstop.js

```diff
@@ -0,0 +1,94 @@
+/* eslint no-underscore-dangle: "off" */
+/* eslint max-len: "off" */
+/* eslint func-names: "off" */
+/* eslint consistent-return: "off" */
+
+// const path = require('path');
+// const _ = require('lodash');
+const mongoose = require('mongoose');
+const express = require('express');
+const bodyParser = require('body-parser');
+const config = require('../../config/index');
+const publisher = require('../../utils/publish');
+
+const storage = require(`./${config.storage}`); // eslint-disable-line
+
+const jsonParser = bodyParser.json();
+// const urlParser = bodyParser.urlencoded({ extended: false });
+const router = express.Router();
+
+const log = require('../../config/logger'); // eslint-disable-line
+
+
+// Start a flow
+router.post('/:id/start', jsonParser, async (req, res) => {
+  const flowId = req.params.id;
+  const credentials = res.locals.credentials[1];
+  let flow;
+
+  if (!mongoose.Types.ObjectId.isValid(flowId)) {
+    return res.status(400).send('Invalid id');
```

Comment:
The API respond sometimes with content type "application/json", sometimes with "text/plain", which is weird. But this is a general problem, not only this PR.

Reply:
That's true. Do we have a shared best practice on how to format the content of such error messages?

Reply:
I'm not sure, that we have such best practices defined. @philecs do we?

Reply:
I don't think we have one but we should define something in our API specs guidelines.

Any preferences? I would prefer `application/json`

Reply:
I think JSON is the option that the IAM is currently using as well (single object with a `msg` key), so we might as well standardise that.

Reply:
I'll set up an issue to add this one to the guidelines.